tools/gluten-it/README.md

```diff
@@ -14,14 +14,17 @@ https://github.com/oap-project/gluten
 
 See official Gluten build guidance https://github.com/oap-project/gluten#how-to-use-gluten
 
-### 2. Install and run gluten-it
+### 2. Install and run gluten-it with Spark version
+
+> Note: Support Spark 3.2 and Spark 3.3 only
```

Comment:
Seems we don't have a place to maintain supported Spark versions, OS and HW. Is it good to centralize it in README?

Reply:
ok, will add in another doc update PR together with how to user header checker and how to download pre-build jar.